code/API_definitions/connectivity-insights.yaml

```
@@ -556,6 +204,50 @@ components:
             current date and time is used and network data for the monitoring
             data aggregation is used to check network performance against the
             policy defined.
```

I've seen the line because of the word "policy". But on a second look: shouldn't the time stamp be part of the response, not the request?

```suggestion
            current date and time is used and network data for the monitoring
            data aggregation is used to check network performance against the
            application profile defined.
```

---

the complete description for the parameter is provided below:
"this is an optional input parameter. A future data and time can be provided for predictive data. If no value is provided then the current date and time is used and network data for the monitoring data aggregation is used to check network performance against the policy defined."

As per the description this parameter is optional field and can be used to provide a future data and time for querying the predictive KPIs from the network.  Hence the timestamp input is valid as part of the request body.

i will go ahead and update the term policy to application profile as rightly recommended in the review. But please note that there are other places in the yaml where the term policy is used and that reflects the individual network KPI thresholds requested by app developer. i will put in a new enhancement issue created to see if we can revisit the term and see if we can find a better developer friendly word.